karma.conf.js

```diff
@@ -0,0 +1,84 @@
+// Karma configuration
```

Comment:
We've changed this file significantly enough that I think we should clean it up.

Reply:
Would you propose removing the comments, superfluous lines, and alphabetizing the options?

Reply:
I would say to remove comments that don't really add any clarity (which looks to be all of the ones that are in here), superfluous lines (like collapsing multiple empty lines into one), but I don't think we need to alphabetize the options as long as the order makes sense.